corehq/apps/reports/filters/forms.py

```diff
@@ -434,8 +434,10 @@ def has_selections(cls, request):
         if instance._show_unknown:
             return True
         for param in params:
-            if param['slug'] in [PARAM_SLUG_APP_ID, PARAM_SLUG_STATUS]:
+            if param['slug'] == PARAM_SLUG_APP_ID:
                 return True
+        if request.GET.get('show_advanced') == 'on':
```

Comment:
nit: it looks like the `FormsByApplicationFilterParams` is intended to be used for parsing these params - you might bump this logic there.

Reply:
Punting on this - request doesn't look like it's very accessible to `FormsByApplicationFilterParams`

Reply:
`FormsByApplicationFilterParams` parses the return value of `cls.get_value(request, domain)`, which is supposed to extract the relevant params from the request.  However, looking over this code a bit, it looks like `show_advanced` is tacked on in an unorthodox way, so I'm not certain it'll actually appear in the `get_value` result (unlike the params defined in `get_labels`).  Seems like a bit of a quagmire that I wouldn't want to get involved in.  Fine by me to drop.